app/assets/stylesheets/helpers/_header.scss

```diff
@@ -497,81 +498,81 @@ $covid-green: #7eff8e;
       color: govuk-colour("black");
     }
   }
+}
 
-  .global-bar-message-container {
-    @include core-19;
-    @extend %site-width-container;
-    position: relative;
-    color: govuk-colour("white");
+.global-bar-message {
+  margin-bottom: 0;
+  margin-top: 0;
+  padding: govuk-spacing(4) 0;
+}
 
-    .global-bar-message {
-      margin-bottom: 0;
-      margin-top: 0;
+.global-bar-title {
+  font-weight: 700;
+  margin-right: govuk-spacing(2);
+  margin-bottom: govuk-spacing(1);
 
-      .global-bar-title {
-        font-weight: 700;
-        margin-right: govuk-spacing(2);
-        margin-bottom: govuk-spacing(1);
+  &:only-child {
+    margin: 0;
+  }
+}
 
-        &:only-child {
-          margin: 0;
-        }
-      }
-    }
+.global-bar-dismiss-wrapper {
+  margin-top: govuk-spacing(4);
+}
 
-    .global-bar__dismiss {
-      display: none;
-    }
+.global-bar__dismiss {
+  display: none;
+}
 
-    .global-bar__dismiss--show {
-      display: inline-block;
-      text-align: right;
-      margin-top: 0;
-    }
-  }
+.global-bar__dismiss--show {
+  display: inline-block;
+}
 
-  .global-bar-additional {
-    display: none;
-    position: relative;
-    margin-top: govuk-spacing(4);
-    margin-bottom: govuk-spacing(4);
+.gem-c-govspeak .global-bar__heading {
```

Comment:
Is it possible to not use a gem class outside of the gem? (I'm having a thing about isolation at the moment, pun not intended)

Reply:
@andysellick The govspeak styles were overriding the `.global-bar__heading` styles without this - I had to add `.gem-c-govspeak` to make it specific enough 😞

Reply:
Okay, fair enough.